ambari-admin/src/main/resources/ui/admin-web/bower.json

```diff
@@ -19,8 +19,5 @@
     "chai": "1.8.0",
     "mocha": "1.14.0",
     "sinon": "1.10.3"
-  },
```

Comment:
not sure, why this file were modified in scope of current task

Reply:
that was by mistake. My bad.

Thanks for taking this up.